handlebars.js

```diff
@@ -1,22 +1,13 @@
 const Handlebars = require('handlebars');
-const {passing, failed, pending} = require('./constants');
+const {statusMarks, pending} = require('./constants');
 const {formatJSON} = require('./files');
 
 // helpersFile is a path to a project specific helpers file.
 const registerHelpers = (helpersFile) => {
```

Comment:
```suggestion
const registerHelpers = helpersFile => {
```

and the function is missing semi-colon after the closing curly braces. (Looks like you'll need to add a lint script)

Reply:
Same changes below for `makeTemplate()` function

Reply:
should be addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/4585e53dd271bcf777588d558747dd6eb3366102